test/note_test.rb

```diff
@@ -5,16 +5,56 @@ def setup
     @frequency   = 220.to_f
     @duration    = 2.0
     @sample_rate = 22050
+    @total_frames = (@duration * @sample_rate).to_i
+    @cycles_per_frame = @frequency / @sample_rate
+    @sine_wave_cycle = 2 * Math::PI * @cycles_per_frame
+    @samples = samples_helper
     @note = MusicTheory::Note.new(frequency: @frequency, duration: @duration)
+    @note_2 = MusicTheory::Note.new(frequency: @frequency, duration: @duration, distort: true)
   end
 
   def test_total_frames
-    frames = (@duration * @sample_rate).to_i
-    assert_equal frames, @note.total_frames
+    assert_equal @total_frames, @note.total_frames
   end
 
   def test_cycles_per_frame
-    cycles_per_frame = @frequency / @sample_rate
-    assert_equal cycles_per_frame, @note.cycles_per_frame
+    assert_equal @cycles_per_frame, @note.cycles_per_frame
+  end
+
+  def test_sine_wave_cycle
+    assert_equal @sine_wave_cycle, @note.sine_wave_cycle
+  end
+
+  def test_samples
+    assert_equal @samples, @note.samples
+  end
+
+  def test_distort!
+    assert_equal distort_helper, @note_2.samples
```

Comment:
Remember to name your variables with descriptive names e.g. `distorted_note` instead of `note_2`